datacats/scripts/compile_less.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-lessc $1 > $2
+
+
+lessc /project/target/ckan/ckan/public/base/less/main.less \
+> /project/target/ckan/ckan/public/base/css/main.debug.css
```

Comment:
Huh, well we could just do a copy, but that would imply that the non-minified sources are good for prod :stuck_out_tongue: